src/main/java/com/gnerga/app/remote/GitHubClient.java

```diff
@@ -0,0 +1,22 @@
+package com.gnerga.app.remote;
+
+import com.gnerga.app.dto.GitHubDetailsDto;
+import com.gnerga.app.remote.model.GitHubResponseDto;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import java.util.List;
+
+@FeignClient(
+        value = "github-client",
+        url = "api.github.com"
```

Comment:
w sytuacji kiedy dajemy urla do clienta dobrze pomyslec o tym zeby byla to zmienna konfigurowalna -> application.properties dzieki temu bedziemy mogli zmieniac w zaleznosci od srodowiska tudziez nawet w celach testowych